examples/medplum-patient-intake-demo/data/core/valuesets.json

```diff
@@ -0,0 +1,182 @@
+            { "system": "https://www.usps.com/", "code": "OK", "display": "Oklahoma" },
+            { "system": "https://www.usps.com/", "code": "OR", "display": "Oregon" },
+            { "system": "https://www.usps.com/", "code": "PW", "display": "Palau" },
+            { "system": "https://www.usps.com/", "code": "PA", "display": "Pennsylvania" },
+            { "system": "https://www.usps.com/", "code": "PR", "display": "Puerto Rico" },
+            { "system": "https://www.usps.com/", "code": "RI", "display": "Rhode Island" },
+            { "system": "https://www.usps.com/", "code": "SC", "display": "South Carolina" },
+            { "system": "https://www.usps.com/", "code": "SD", "display": "South Dakota" },
+            { "system": "https://www.usps.com/", "code": "TN", "display": "Tennessee" },
+            { "system": "https://www.usps.com/", "code": "TX", "display": "Texas" },
+            { "system": "https://www.usps.com/", "code": "UT", "display": "Utah" },
+            { "system": "https://www.usps.com/", "code": "VT", "display": "Vermont" },
+            { "system": "https://www.usps.com/", "code": "VI", "display": "Virgin Islands" },
+            { "system": "https://www.usps.com/", "code": "VA", "display": "Virginia" },
+            { "system": "https://www.usps.com/", "code": "WA", "display": "Washington" },
+            { "system": "https://www.usps.com/", "code": "WV", "display": "West Virginia" },
+            { "system": "https://www.usps.com/", "code": "WI", "display": "Wisconsin" },
+            { "system": "https://www.usps.com/", "code": "WY", "display": "Wyoming" },
+            {
+              "system": "https://www.usps.com/",
+              "code": "AE",
+              "display": "Armed Forces Europe, the Middle East, and Canada"
+            },
+            { "system": "https://www.usps.com/", "code": "AP", "display": "Armed Forces Pacific" },
+            { "system": "https://www.usps.com/", "code": "AA", "display": "Armed Forces Americas (except Canada)" }
+          ]
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:183a569a-7f6d-41f9-97d8-57691fbc2876",
+      "request": {
+        "method": "POST",
+        "url": "ValueSet?name=smoking-status",
+        "ifNoneExist": "url=http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.11.20.9.38"
+      },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "version": "0.0.0",
+        "url": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.11.20.9.38",
+        "name": "smoking-status",
+        "title": "Smoking Status",
+        "expansion": {
+          "timestamp": "2024-08-12T00:00:00.000Z",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/valueset-expansionSource",
+              "valueUri": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.11.20.9.38"
+            }
+          ],
```

Comment:
@rahul1 This addresses the Smoking Status ValueSet https://github.com/medplum/medplum/pull/5049#discussion_r1713653164